.github/workflows/push-1password-secrets-to-azure.yaml

```diff
@@ -0,0 +1,51 @@
+name: "Push secrets from 1Password to Azure"
+
+permissions:
+  id-token: write
+  contents: read
+
+on:
+  push:
+    branches:
+      - "azure-push-secrets-*"
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: false
+
+jobs:
+  push-azure-secrets:
+    name: "Push Azure Secret"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Install 1Password CLI
+        uses: 1password/install-cli-action@143a85f84a90555d121cde2ff5872e393a47ab9f # v1.0.0
+
+      - name: Load Azure token
+        id: op-azure-token
+        uses: 1password/load-secrets-action@581a835fb51b8e7ec56b71cf2ffddd7e68bb25e0 # v2.0.0
+        with:
+          export-env: false
+        env:
+          OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
```

Comment:
Are there docs on how this token was made?

Do we need to add setting it to the GitHub secrets pulimi config?

Reply:
> Are there docs on how this token was made?

There are notes the 1password secret on how to recreate this. Is there another, better place to put those notes?

> Do we need to add setting it to the GitHub secrets pulimi config?

Good idea. I'll address that in a separate PR

Reply:
Nope that is the best spot. I now realize you may have mentioned that the notes are in 1password before. Sorry if I missed that and thanks!